app/build.gradle

```diff
@@ -71,6 +71,7 @@ dependencies {
     implementation 'jp.wasabeef:picasso-transformations:2.2.1'
     implementation 'com.google.android.exoplayer:exoplayer:2.9.6'
     implementation 'com.github.rustamg:file-dialogs:1.0'
+    implementation group: 'info.debatty', name: 'java-string-similarity', version: '1.2.1'
```

Comment:
Small nitpick: better to be consistent in the naming with other items.

Reply:
Oops, yes.